src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/ReduceTokensCodeCleanupProvider.vb

```diff
@@ -242,6 +287,17 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
 
                 Return token.CopyAnnotationsTo(SyntaxFactory.DecimalLiteralToken(leading, newValueString, token.GetTypeCharacter(), newValue, trailing))
             End Function
+
+            Private Shared Function CreateIntegerLiteralToken(token As SyntaxToken, newValueString As String, newValue As ULong) As SyntaxToken
+                Debug.Assert(token.Kind = SyntaxKind.IntegerLiteralToken)
+
+                ' create a new token with valid token text and carries over annotations attached to original token to be a good citizen 
+                ' it might be replacing a token that has annotation injected by other code cleanups
+                Dim leading = If(token.LeadingTrivia.Count > 0, token.LeadingTrivia, SyntaxTriviaList.Create(SyntaxFactory.ElasticMarker))
```

Comment:
Now that we have three copies of this logic, can we refactor this with the methods above?